File: Lab5/Domain/Entities/RentalRecord.cs
namespace Lab5.Domain.Entities;

public sealed class RentalRecord
{
    public RentalRecord(Guid id, Guid userId, Guid bookId, DateOnly rentedOn, DateOnly dueDate)
    {
        Id = id;
        UserId = userId;
        BookId = bookId;
        RentedOn = rentedOn;
        DueDate = dueDate;
    }

    public Guid Id { get; }
    public Guid UserId { get; }
    public Guid BookId { get; }
    public DateOnly RentedOn { get; }
    public DateOnly DueDate { get; }
}

File: Lab5.Web/src/domain/entities/RentalRecord.js
export class RentalRecord {
  constructor(id, userId, bookId, rentedOn, dueDate) {
    this.id = id;
    this.userId = userId;
    this.bookId = bookId;
    this.rentedOn = rentedOn;
    this.dueDate = dueDate;
  }
}
